blueprint/core/blueprint_ImageView.h

```diff
@@ -14,6 +14,20 @@
 
 namespace blueprint
 {
+    namespace
+    {
+        // juce::URL::isWellFormed is currently not a complete
+        // implementation, so we have this slightly more robust check
+        // for now.
+        bool isWellFormedURL(const juce::URL &url)
+        {
+            return url.isWellFormed()  &&
+                   url.getScheme().isNotEmpty() &&
+                   !url.toString(false).startsWith("data");
```

Comment:
Heh, the `juce::URL` class is confusing, because a `data:` URL _is_ well formed!